src/SFML/Graphics/Sprite.cpp

```diff
@@ -149,29 +144,21 @@ void Sprite::draw(RenderTarget& target, const RenderStates& states) const
 ////////////////////////////////////////////////////////////
 void Sprite::updatePositions()
 {
-    FloatRect bounds = getLocalBounds();
-
-    m_vertices[0].position = Vector2f(0, 0);
-    m_vertices[1].position = Vector2f(0, bounds.height);
-    m_vertices[2].position = Vector2f(bounds.width, 0);
-    m_vertices[3].position = Vector2f(bounds.width, bounds.height);
+    FloatRect bounds(getLocalBounds());
```

Comment:
```suggestion
    FloatRect bounds = getLocalBounds();
```
Unnecessary style change.